### DISCUSSION TITLE
demo.netbox.dev --> User Admin was locked

### DISCUSSION THREAD

**Discussion by LHBL2003** [netbox-community/netbox#11940] [Help Wanted!] [answered]
### NetBox version

v3.4.5

### Python version

3.8

### Steps to Reproduce

Someone has locked out the admin at https://demo.netbox.dev/.  @jeremystretch  Can you unlock it?

### Expected Behavior

-

### Observed Behavior

-

**Comment by candlerb** [comment]
To clarify, "locked out" means "can login, but has no permissions" (i.e. I suspect the "superuser" privilege has been removed)

**Comment by jeremystretch** [comment, marked as answer]
The demo has been rebuilt and the admin login works again.
